x-pack/plugins/security/public/management/api_keys/documentation_links.ts

```diff
@@ -7,17 +7,19 @@
 import { DocLinksStart } from 'src/core/public';
 
 export class DocumentationLinksService {
-  private readonly esDocBasePath: string;
+  private readonly apiKeySettings: string;
+  private readonly createApiKey: string;
 
   constructor(docLinks: DocLinksStart) {
-    this.esDocBasePath = `${docLinks.ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/${docLinks.DOC_LINK_VERSION}/`;
+    this.apiKeySettings = `${docLinks.links.security.apiKeyServiceSettings}`;
```

Comment:
>    this.apiKeySettings = `${docLinks.links.security.apiKeyServiceSettings}`;

If it's preferrable to remove these definitions and use the doc links service directly in the subsequent functions, I can make that change.

Reply:
Yeah, if you can use the links service directly that would be great, thanks!

Reply:
Actually I think we don't need our own `DocumentationLinksService`'s anymore. But if it's too much for this PR then I'm fine to keep it as is and we'll remove these services in the follow up.

Reply:
Sure, if we do the cleanup in a subsequent PR that works for me!